### DISCUSSION TITLE
Resource not load from the package

### DISCUSSION THREAD

**Discussion by rmitesh** [filamentphp/filament#7029] [Help] [answered]
I am developing a package, where I have used resource.

I have added `$resources` but in the it doesn't appear in the sidebar.
```php
use Rmitesh\MyPackage\Resources\CardResource;

class MyPackageServiceProvider extends PackageServiceProvider
{
    protected array $resources = [
        CardResource::class,
    ];
}
```

![image](https://github.com/filamentphp/filament/assets/48554454/83a89f20-b1c7-4573-bdbc-beeea6da6f60)

I have tried `livewire:discover`  `filament:upgrade` and `composer dump-autoload` all commands, but it's working.
I'm missed any steps?

**Comment by howdu** [comment]
- Can you see the resource route when you run `route:list`?
- Do you have a policy on the model?
- Check if your service provider is getting registered by Laravel by adding a simple output. 
```php
public function bootingPackage()
{
  dd('it works');
}
```
If it's not then check your composer.json has extra/laravel/providers.
- Also presume CardResource doesn't have any special code for disabling the navigation item.

**Comment by rmitesh** [comment, marked as answer]
Silly me, actually I will have to extends Filament's `Filament\PluginServiceProvider`

It will works 🥂
